sdl/_testdata/deployment-svc-mismatch.yaml

```diff
@@ -0,0 +1,45 @@
+---
+version: "2.0"
```

Comment:
What is the error in this file that leads to the test case? I don't see it immediately.